apis/python/operator/src/lib.rs

```diff
@@ -81,15 +81,33 @@ impl PyEvent {
     pub fn to_py_dict(self, py: Python<'_>) -> PyResult<Py<PyDict>> {
         let mut pydict = HashMap::new();
         match &self.event {
-            MergedEvent::Dora(_) => pydict.insert("kind", "dora".to_object(py)),
-            MergedEvent::External(_) => pydict.insert("kind", "external".to_object(py)),
+            MergedEvent::Dora(_) => pydict.insert(
+                "kind",
+                match "dora".into_pyobject(py) {
```

Comment:
It seems to me that you can just create a PyString::from "dora" here. No need for so much complexity.